docs/source/reader_and_writer.rst

```diff
@@ -11,188 +11,102 @@ so you can keep your code agnostic to the storages you use.
-+---------------+--------------------------------------------------------------------------------------------------+
-
-.. note::
-
-    Sparkly doesn't contain any jars inside, so you will have to take care of this.
-    Java connectors for mysql could be found on https://dev.mysql.com/downloads/connector/j/.
-    We usually place them within our service/package codebase in `resources` directory.
-    It's not the best idea to place binaries within a source code, but it's pretty convenient.
-
-.. code-block:: python
-
-    from sparkly import SparklyContext
-    from sparkly.utils import absolute_path
-
-
-    class MyContext(SparklyContext):
-        # Feel free to play with other versions.
-        jars = [absolute_path(__file__, './path/to/mysql-connector-java-5.1.39-bin.jar')]
-
-    hc = MyContext()
-
-    # To read data
-    df = hc.read_ext.mysql('localhost', 'my_database', 'my_table',
-                           options={'user': 'root', 'password': 'root'})
-    # To write data
-    df.write_ext.mysql('localhost', 'my_database', 'my_table', options={
-        'user': 'root',
-        'password': 'root',
-        'rewriteBatchedStatements': 'true',  # improves write throughput dramatically
-    })
-
 .. _kafka:
 
 Kafka
-^^^^^
+-----
 
 Sparkly's reader and writer for Kafka are built on top of the official spark package
 for Kafka and python library `kafka-python <https://github.com/dpkp/kafka-python>`_ .
 The first one allows us to read data efficiently,
 the second covers a lack of writing functionality in the official distribution.
 
-+---------------+--------------------------------------------------------------------------------------------+
-| Package       | https://mvnrepository.com/artifact/org.apache.spark/spark-streaming-kafka_2.10             |
-+---------------+--------------------------------------------------------------------------------------------+
-| Configuration | http://spark.apache.org/docs/latest/streaming-kafka-0-10-integration.html                  |
-+---------------+--------------------------------------------------------------------------------------------+
++---------------+------------------------------------------------------------------------------------------+
+| Package       | https://mvnrepository.com/artifact/org.apache.spark/spark-streaming-kafka-0-8_2.11/2.1.0 |
++---------------+------------------------------------------------------------------------------------------+
+| Configuration | http://spark.apache.org/docs/2.1.0/streaming-kafka-0-8-integration.html                  |
++---------------+------------------------------------------------------------------------------------------+
 
 .. note::
-    - To use the Kafka functionality **sparkly** needs the **kafka-python** library which is an optional dependency.
-      So you need to install **sparkly** with **kafka** extras:
+    - To interact with Kafka ``sparkly`` needs ``kafka-python`` library, you can get it via:
```

Comment:
Grammar is just a little bit off:
To interact with Kafka, ``sparkly`` needs the ``kafka-python`` library.  You can get it via: